srcpkgs/gnome-builder/template

```diff
@@ -21,7 +21,7 @@ short_desc="IDE for GNOME"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://wiki.gnome.org/Apps/Builder"
-changelog="https://gitlab.gnome.org/GNOME/gnome-builder/-/raw/gnome-builder-44/NEWS"
+changelog="https://gitlab.gnome.org/GNOME/gnome-builder/-/raw/main/NEWS"
```

Comment:
This should be `https://gitlab.gnome.org/GNOME/gnome-builder/-/raw/gnome-builder-46/NEWS`